### DISCUSSION TITLE
UBL only probes 57 of 100 points?

### DISCUSSION THREAD

**Discussion by Punisher187** [Jyers/Marlin#240] [Q&A] [answered]
I have adjusted my x,y,z offsets and run ABL it probes 57 of 100 points then stops and prompts me to save. It does the same if I do it from the machine screen or if I do it through OctoPrint. My mesh shows up and I'm getting good prints but I'm stuck thinking I'm doing something wrong?

Specs:
Ender3 v2
BLTouch 3.1
Using the pre configured 1.2.1, E3V2-UBL-BLTouch-10x10-v4.2.2.bin firmware.
OctoPrint on a Pi

**Comment by kentsangcanada** [comment]
The same happens to me when i am complete I am at 157/225

**Comment by ElieLiabeuf** [comment]
Same here, can't understand why...

Edits: Seems normal after all. It displays all the point in the mesh and only probe the accessible ones.

**Comment by Schnabulation** [comment, marked as answer]
We already had that discussion: the probe can not reach all 100 points and therefore only probes the one it can reach. The other points can be extrapolated with `G29 P3` I believe...

**Reply by Jyers** [reply]
As a note: The firmware handles all required extrapolation for you using G29 P3 commands
